test/cases/ostree-simplified-installer.sh

```diff
@@ -279,6 +279,24 @@ sudo ostree --repo="$PROD_REPO" remote add --no-gpg-verify edge-stage "$STAGE_RE
 greenprint "🔧 Prepare stage repo network"
 sudo podman network inspect edge >/dev/null 2>&1 || sudo podman network create --driver=bridge --subnet=192.168.200.0/24 --gateway=192.168.200.254 edge
 
+###########################################################
+##
+## Prepare fdo server
+##
+###########################################################
+greenprint "🔧 Prepare fdo manufacturing server"
+sudo git clone https://github.com/runcom/fdo-containers
```

Comment:
I wonder if we should create a tag and then pin the tag here. We like to test with/against versions of stuff.

Reply:
or even better: just a fixed commit (tags can change)

Reply:
I’m ok with that ofc - we’re gonna switch the create keys script with a rust cli command in fdo in the next release anyway and we’re probably adding a command to generate configs too (soon), for now I think pinning the commit works